client/src/tokens/colorVariables.test.js

```diff
@@ -0,0 +1,114 @@
+const colors = require('./colors');
+const { generateColorVariables } = require('./colorVariables');
+
+describe('generateColorVariables', () => {
+  it('generates all variables', () => {
+    const colorVariables = generateColorVariables(colors);
+    const generatedVariables = Object.keys(colorVariables);
+    Object.values(colors).forEach((hues) => {
+      Object.values(hues).forEach((shade) => {
+        expect(generatedVariables).toContain(shade.cssVariable);
+      });
+    });
+  });
+
+  it('is stable (update the docs when this changes)', () => {
```

Comment:
I love the idea of this test – but what part of the docs, exactly, should be updated when this changes? A comment here for people other than you who might encounter a failure of this test to explain what to update exactly would be nice.

Reply:
Good point! I’ve added a fair bit.